Update device-control-removable-storage-access-control.md (#9287)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/device-control-removable-storage-access-control.md

|Property Name|Description|Options|
|---|---|---|
|**GroupId**|GUID, a unique ID, represents the group and will be used in the policy.||

> |**DescriptorIdList**|List the device properties you want to use to cover in the group. All properties are case sensitive. |**PrimaryId**: The Primary ID includes `RemovableMediaDevices`, `CdRomDevices`, `WpdDevices`, `PrinterDevices`. <p>**InstancePathId**: InstancePathId is a string that uniquely identifies the device in the system, for example, `USBSTOR\DISK&VEN_GENERIC&PROD_FLASH_DISK&REV_8.07\8735B611&0`, `Device instance path` in the Device Manager. The number at the end (for example &0) represents the available slot and may change from device to device. For best results, use a wildcard at the end. For example, `USBSTOR\DISK&VEN_GENERIC&PROD_FLASH_DISK&REV_8.07\8735B611*`. <p>**DeviceId**: Transform `Device instance path` to Device ID format, see [Standard USB Identifiers](/windows-hardware/drivers/install/standard-usb-identifiers), for example, USBSTOR\DISK&VEN_GENERIC&PROD_FLASH_DISK&REV_8.07 <p>**HardwareId**: A string identified the device in the system, for example, USBSTOR\DiskGeneric_Flash_Disk___8.07, `Hardware Ids` in the Device Manager; Note: Hardware ID is not unique; different devices might share the same value.<p>**FriendlyNameId**: A string attached to the device, for example, Generic Flash Disk USB Device, `Friendly name` in the Device Manager. <p>**BusId**: For example, USB, SCSI <p>**SerialNumberId**: You can find SerialNumberId from `Device instance path` in the Device Manager, for example, `03003324080520232521` is SerialNumberId in USBSTOR\DISK&VEN__USB&PROD__SANDISK_3.2GEN1&REV_1.00\\`03003324080520232521`&0 <p>**VID_PID**: Vendor ID is the four-digit vendor code that the USB committee assigns to the vendor. Product ID is the four-digit product code that the vendor assigns to the device; Support wildcard, transform `Device instance path` to Vendor ID and Product ID format, see [Standard USB Identifiers](/windows-hardware/drivers/install/standard-usb-identifiers). <p>`0751_55E0`: match this exact VID/PID pair<p>`_55E0`: match any media with PID=55E0 <p>`0751_`: match any media with VID=0751 <p> **Note**: Look at the 'How can I find the media property in the Device Manager?' under 'Fequently asked questions' section below to understand how to find the property in Device Manager.|

Reviewer: ```suggestion
|**DescriptorIdList**|List the device properties you want to use to cover in the group. All properties are case sensitive. |**PrimaryId**: The Primary ID includes `RemovableMediaDevices`, `CdRomDevices`, `WpdDevices`, and `PrinterDevices`. <p>**InstancePathId**: InstancePathId is a string that uniquely identifies the device in the system, for example, `USBSTOR\DISK&VEN_GENERIC&PROD_FLASH_DISK&REV_8.07\8735B611&0`, `Device instance path` in the Device Manager. The number at the end (for example &0) represents the available slot and may change from device to device. For best results, use a wildcard at the end. For example, `USBSTOR\DISK&VEN_GENERIC&PROD_FLASH_DISK&REV_8.07\8735B611*`. <p>**DeviceId**: Transforms `Device instance path` to Device ID format, see [Standard USB Identifiers](/windows-hardware/drivers/install/standard-usb-identifiers), for example, USBSTOR\DISK&VEN_GENERIC&PROD_FLASH_DISK&REV_8.07. <p>**HardwareId**: A string identified the device in the system, for example, USBSTOR\DiskGeneric_Flash_Disk___8.07, `Hardware Ids` in the Device Manager. Note: Hardware ID is not unique; different devices might share the same value.<p>**FriendlyNameId**: A string attached to the device, for example, Generic Flash Disk USB Device, `Friendly name` in the Device Manager. <p>**BusId**: For example, USB, SCSI. <p>**SerialNumberId**: You can find SerialNumberId from `Device instance path` in the Device Manager, for example, `03003324080520232521` is SerialNumberId in USBSTOR\DISK&VEN__USB&PROD__SANDISK_3.2GEN1&REV_1.00\\`03003324080520232521`&0. <p>**VID_PID**: Vendor ID is the four-digit vendor code that the USB committee assigns to the vendor. Product ID is the four-digit product code that the vendor assigns to the device; Support wildcard, transform `Device instance path` to Vendor ID and Product ID format, see [Standard USB Identifiers](/windows-hardware/drivers/install/standard-usb-identifiers). <p>`0751_55E0`: match this exact VID/PID pair<p>`_55E0`: match any media with PID=55E0 <p>`0751_`: match any media with VID=0751 <p> **Note**: Look at the 'How can I find the media property in the Device Manager?' under the 'Fequently asked questions' section that follows to understand how to find the property in Device Manager.|
```

|summarize dcount(DeviceName) by PlatformVersion // check how many machines are using which platformVersion
|order by PlatformVersion desc
```

### How can I find the media property in the Device Manager?

> 1. Plugin the media

Reviewer: ```suggestion
1. Plug in the media.
```

> 2. Open `Device Manager`

Reviewer: ```suggestion
2. Open `Device Manager`.
```

![image](https://user-images.githubusercontent.com/81826151/181859412-affd6aa1-09ad-44bf-9541-330499cc2c87.png)

> 3. Locate the media in the Device Manager, right-click and select `Properties`

Reviewer: ```suggestion
3. Locate the media in the Device Manager, then right-click and select `Properties`.
```

![image](https://user-images.githubusercontent.com/81826151/181859700-62a6f704-b12e-41e3-a048-7d63432654a4.png)

> 4. Open `Details`, and select `Property`

Reviewer: ```suggestion
4. Open `Details`, and select `Property`.
```
